src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Logging/LspServiceLogger.cs

```diff
@@ -35,7 +35,7 @@ public LspServiceLogger(ILogger hostLogger)
     /// <summary>
     /// TODO - Switch this to call LogInformation once appropriate callers have been changed to LogDebug.
     /// </summary>
-    public override void LogInformation(string message, params object[] @params) => _hostLogger.LogDebug(message, @params);
+    public override void LogInformation(string message, params object[] @params) => _hostLogger.LogInformation(message, @params);
```

Comment:
we were previously downgrading all info logs to debug in vscode because information logs were too verbose.  That was because originally the lowest LSP logger level was info, and we ended up logging everything as info.

this fixes it by logging info as info, and adjusting the usages of info logging to a more appropriate level.

Reply:
Oh thank you for finally fixing this!